src/components/explorer/searchResultsTables/ExperimentsTable.js

```diff
@@ -28,59 +28,67 @@ ExperimentRender.propTypes = {
     }).isRequired,
 };
 
-const SEARCH_RESULT_COLUMNS_EXP = [
-    {
-        title: "Experiment",
-        dataIndex: "experimentId",
-        render: (experimentId, record) => <ExperimentRender experimentId={experimentId} {...record} />,
-        sorter: (a, b) => a.experimentId.localeCompare(b.experimentId),
-        defaultSortOrder: "ascend",
-    },
-    {
-        title: "Individual",
-        dataIndex: "individual",
-        render: (individual) => <>{individual.id}</>,
-        sorter: (a, b) => a.individual.id.localeCompare(b.individual.id),
-        sortDirections: ["descend", "ascend", "descend"],
-    },
-    {
-        title: "Biosample",
-        dataIndex: "biosampleId",
-        render: (biosampleId, record) => (
-            <BiosampleIDCell biosample={biosampleId} individualID={record.individual.id} />
-        ),
-        sorter: (a, b) => a.biosampleId.localeCompare(b.biosampleId),
-        sortDirections: ["descend", "ascend", "descend"],
-    },
-    {
-        title: "Study Type",
-        dataIndex: "studyType",
-        render: (studyType) => <>{studyType}</>,
-        sorter: (a, b) => a.studyType.localeCompare(b.studyType),
-        sortDirections: ["descend", "ascend", "descend"],
-    },
-    {
-        title: "Experiment Type",
-        dataIndex: "experimentType",
-        render: (expType) => <>{expType}</>,
-        sorter: (a, b) => a.experimentType.localeCompare(b.experimentType),
-        sortDirections: ["descend", "ascend", "descend"],
-    },
-];
-
 const ExperimentsTable = ({ data, datasetID }) => {
     const tableSortOrder = useSelector(
         (state) => state.explorer.tableSortOrderByDatasetID[datasetID]?.["experiments"],
     );
 
+    const experimentTypeFilters = useDynamicTableFilterOptions(data, "experimentType");
+
+    const columns = useMemo(() => {
+        const baseColumns = [
```

Comment:
just return this array directly instead of assigning it to a variable which immediately gets returned.